code/modules/antagonists/eldritch_cult/eldritch_magic.dm

```diff
@@ -47,6 +47,10 @@
 	inhand_icon_state = "mansus"
 	catchphrase = "R'CH T'H TR'TH"
 
+/obj/item/melee/touch_attack/mansus_fist/ignition_effect(atom/A, mob/user)
+	. = "<span class='notice'>[user] effortlessly snaps [user.p_their()] fingers near [A], igniting it with eldritch energies. Fucking badass!</span>"
+	qdel(src)
```

Comment:
maybe this should give you a partial refund on the cooldown of the spell, since this is a relatively minor effect?

Reply:
Dude

Reply:
the cooldown is already like 10 seconds, no.